inst/WORDLIST

```diff
@@ -1,5 +1,19 @@
+Aarhus
 CMD
-DuckDB
-Stata
+HbA
+HbA1c
+Isaksen's
+Lifecycle
+Metformin
+ORCID
+OSCD
```

Comment:
It's OSDC

It's too bad that one algorithm ends with DC while the other (RSCD) ends with CD :)